libbpf-tools/Makefile

```diff
@@ -158,6 +158,18 @@ ifeq ($(USE_BLAZESYM),1)
 LDFLAGS += $(OUTPUT)/libblazesym.a -lrt -lpthread -ldl
 endif
 
+# In sigsnoop, SIGRTMAX and SIGRTMIN are used as array indexes. However,
+# SIGRTMAX and SIGRTMIN are not constants in glibc, so we add constants.
```

Comment:
seems glibc is not totally obey the POSIX rule

![image](https://github.com/user-attachments/assets/799b9067-879b-4e9f-8df0-ed16f1286495)

Reply:
@chenhengqi Hi, hengqi, SIGRTMIN is not compile-time const.